### PR TITLE
Add Examples from Docs to  README

### PR DESCRIPTION
**Pull request by brminnick**
This PR copy/pastes the example found in [https://docs.microsoft.com/dotnet/communitytoolkit/maui/markup/markup](https://docs.microsoft.com/dotnet/communitytoolkit/maui/markup/markup?WT.mc_id=dotnet-0000-bramin) and adds it to the README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,143 @@ public static MauiApp CreateMauiApp()
...
+entry.SetBinding(Entry.TextProperty, new Binding(nameof(ViewModel.RegistrationCode));
+```
+
+Markup allows us to define the binding fluently and therefore chain multiple methods together to reduce the verbosity of our code:
+
+```csharp
+new Entry().Bind(Entry.TextProperty, nameof(ViewModel.RegistrationCode))
+```
+
+For further details on the possible options for the `Bind` method refer to the [`BindableObject` extensions documentation](extensions/bindable-object-extensions.md).
```
Comment (bijington) [inline README.md:48]:
```suggestion
For further details on the possible options for the `Bind` method refer to the [`BindableObject` extensions documentation](https://docs.microsoft.com/dotnet/communitytoolkit/maui/markup/extensions/bindable-object-extensions).
```

File: README.md [hunk 2]
```diff
@@ -26,6 +26,143 @@ public static MauiApp CreateMauiApp()
...
+entry.HeightRequest = 40;
+```
+
+Markup allows us to define the sizing fluently and therefore chain multiple methods together to reduce the verbosity of our code:
+
+```csharp
+new Entry().Size(200, 40);
+```
+
+For further details on the possible options for the `Size` method refer to the [`VisualElement` extensions documentation](extensions/visual-element-extensions.md).
```
Comment (bijington) [inline README.md:66]:
```suggestion
For further details on the possible options for the `Size` method refer to the [`VisualElement` extensions documentation](https://docs.microsoft.com/dotnet/communitytoolkit/maui/markup/extensions/visual-element-extensions).
```

Comment (bijington) [reply README.md:66]: Just fixing the link to the docs site

File: README.md (hunk 1 above)
Comment (bijington) [reply README.md:48]: Just fixing the link to the docs site

**Review by bijington — COMMENTED** [review]
Thanks @brminnick theres just fixing the link to the docs site

**Review by bijington — APPROVED** [review]

**Comment by brminnick** [comment]
Ah - good catch! Thanks Shaun!!!

**Pull request merged**